src/foam/nanos/auth/ProfilePictureView.js

```diff
@@ -200,6 +200,7 @@ foam.CLASS({
     function onRemoveClicked (e) {
       this.dragActive = false;
       this.ProfilePictureImage= null;
+      this.document.querySelector('.attachment-input').value = null;
```

Comment:
What if there's more than one attachment-input on the screen at once? I'll show you a better way to do this.

Reply:
What would be the better solution for this issue? Thanks.